### PR TITLE
Class 09

### PR DESCRIPTION
**Pull request by pjosifovic**

### CODE REVIEW

File: scripts/projectView.js [hunk 1]
```diff
@@ -1,87 +1,90 @@
...
-  $('.main-nav .tab').on('click', function(){
-    var $tabContent = $('.tab-content');
-    var $dataContent = $(this).attr('data-content');
-    $tabContent.hide();
-    $tabContent.filter('#' + $dataContent).fadeIn();
-  });
+  projectView.handleMainNav = function(){
+    $('.main-nav .tab').on('click', function(){
+      var $tabContent = $('.tab-content');
+      var $dataContent = $(this).attr('data-content');
```
Comment (lexm) [inline scripts/projectView.js:7]: This variable is used only once, so it may not be necessary to cache its contents.

File: scripts/script.js [hunk 2]
```diff
@@ -96,3 +66,21 @@
       };
     });
   };
+
+  Project.fetchAll = function(next) {
+    if(localStorage.articles) {
+      Project.loadAll(JSON.parse(localStorage.articles));
+      console.log('if lS exist, after parsing it!');
```
Comment (lexm) [inline scripts/script.js:73]: Is this log statement still necessary?

File: scripts/projectView.js [hunk 3]
```diff
@@ -1,87 +1,90 @@
...
-    // console.log('selected Val ' + $(this).val());
-    if($(this).val()) {
-      var $articles = $('article');
-      var $selectVal = $(this).val();
-      $articles.hide();
-      $articles.each(function(){
-        if($(this).attr('data-title') === $selectVal) {
+  projectView.handleTitleFilter = function() {
+    $('#title-filter').on('change', function () {
+      // console.log('selected Val ' + $(this).val());
```
Comment (lexm) [inline scripts/projectView.js:35]: This commented-out line can go.

File: scripts/projectView.js [hunk 4]
```diff
@@ -1,87 +1,90 @@
...
+        var $selectVal = $(this).val();
+        $articles.hide();
+        $articles.each(function(){
+          if($(this).attr('data-category') === $selectVal) {
+            $(this).fadeIn();
+          }
+        });
+      } else {
+        $('article').each(function() {
           $(this).fadeIn();
```
Comment (lexm) [inline scripts/projectView.js:67]: You can get rid of the .each here: the .fadeIn() method will iterate through the selected elements without it.

File: scripts/projectView.js [hunk 5]
```diff
@@ -1,87 +1,90 @@
-var projectView = {};
```
Comment (lexm) [inline scripts/projectView.js:1]: My comments below are pretty nit-picky, which is a good thing: your code's pretty clean!

File: index.html [hunk 6]
```diff
@@ -68,12 +70,12 @@
...
         </section>
       </section>
-      <section id="blog-stats">
+      <section class="blog-stats">
         <h2>Blog Stats</h2>
         <dt>Total Projects:<dd class="num-projects"></dd></dt>
         <dt>Total Words:<dd class="num-words"></dd></dt>
         <ul class="project-stats">
-          <!-- TODO: Use a handlebars template to fill in some stats for each author -->
+          <!-- blog state handlebars template goes here -->
```
Comment (lexm) [inline index.html:78]: I think you mean "blog stats handlebars template" here.

File: scripts/projectView.js (hunk 4 above)
Comment (pjosifovic) [inline scripts/projectView.js:67]: Not sure if fadeIn() will do that for me. I think i need to keep that .each() so i can itetate thru all articles but I will look into that.

**Pull request merged**